File: 1212-sequential-digits/sequential-digits.cpp
class Solution {
public:
    vector<int> sequentialDigits(int low, int high) {
        vector<int> res;

        for (int start = 1; start <= 9; ++start) {
            int num = start;
            for (int nextDigit = start + 1; nextDigit <= 9; ++nextDigit) {
                num = num * 10 + nextDigit;
                if (num >= low && num <= high) {
                    res.push_back(num);
                }
            }
        }

        sort(res.begin(), res.end());

        return res;
    }
};

File: 1212-sequential-digits/sequential-digits.java
class Solution {
    public List<Integer> sequentialDigits(int low, int high) {
        ArrayList<Integer> res = new ArrayList<Integer>();
        for(int i=1;i<=9;i++){
            int n=i;
            for(int j=i+1;j<=9;++j){
                n=n*10 + j;
                if(n>=low && n<=high){
                    res.add(n);
                }
            }
        }
        Collections.sort(res);
        return res;   
    }
}